zrml/styx/src/weights.rs

```diff
@@ -56,12 +55,12 @@ impl<T: frame_system::Config> WeightInfoZeitgeist for WeightInfo<T> {
     // Storage: Styx Crossings (r:1 w:1)
     // Storage: Styx BurnAmount (r:1 w:0)
     fn cross() -> Weight {
-        Weight::from_ref_time(23_072_000)
+        Weight::from_ref_time(63_231_000)
```

Comment:
This is just the base weight and in addition to that you have all the reads and writes. Think of it like the processing time of the code and the reads and writes are the time to write and read to the storage.

Reply:
The base weight usually represents a constant effort to execute code that is independent from the branches taken and it does not include and storage read or writes. It is just constant computational complexity and storage complexity (variables). Any database complexity and conditional complexity based on parameters is added after the constant weight based on the respective parameters.

This might be helpful: https://docs.substrate.io/test/benchmark/